mimic/model/clb_objects.py

```diff
@@ -38,6 +42,28 @@ def __init__(self):
         self.lbs = {}
         self.meta = {}
 
+    def __len__(self):
```

Comment:
To be clear I would not veto a change based on the use of special methods, but I thought it was important to lay out my thoughts on this so we don't start going too far in this direction.